operator/bundle_helpers/patch-csv.py

```diff
@@ -96,7 +98,7 @@ def parse_args():
                         nargs='+', help='Replacement directives for the RBAC proxy image',
                         default=[])
     parser.add_argument("--add-supported-arch", action='append', required=False,
-                        help='Add label to CSV for an additional operator (may be passed multiple times)',
+                        help='Enable additional operator architecture via CSV labels (may be passed multiple times)',
```

Comment:
"additional" doesn't seem to make sense if we're starting without any.  (Note that if no such labels are declared, then `amd64` is implied, but once *any* flags are, only those specified are enabled.)